lib/airplayer.rb

```diff
@@ -3,9 +3,11 @@
 
 module AirPlayer
   require 'airplayer/version'
-  require 'airplayer/app'
   require 'airplayer/controller'
   require 'airplayer/device'
   require 'airplayer/playlist'
   require 'airplayer/media'
+  require 'airplayer/youtube_dl'
+
+  require 'airplayer/app'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.